mediator/med_io_mod.F90

```diff
@@ -522,36 +526,27 @@ subroutine med_io_write_FB(filename, iam, FB, whead, wdata, nx, ny, nt, &
        return
     endif
 
-    call FB_getFieldN(FB, 1, field, rc=rc)
+    call ESMF_FieldGet(fieldlist(1), mesh=mesh, rc=rc)
```

Comment:
This assumes that all the fields in the field list is in same mesh. Right?

Reply:
Yes that is correct. All fields in a given fieldbundle are assumed to be on the same mesh. That is an underlying assumption in CMEPS.